kratos/python_scripts/testing/run_cpp_mpi_tests.py

```diff
@@ -1,12 +1,27 @@
-import sys
+import argparse
 
 from KratosMultiphysics import mpi, Tester
 
-if __name__ == '__main__':
-    #Tester.SetVerbosity(Tester.Verbosity.TESTS_LIST)
-    Tester.SetVerbosity(Tester.Verbosity.TESTS_OUTPUTS)
+def main():
+    parser = argparse.ArgumentParser()

```

Comment:
how about adding a check for running distributed?

Sth like `assert Kratos.IsDistibutedRun()`